core/src/main/java/org/apache/carbondata/core/scan/executor/infos/BlockExecutionInfo.java

```diff
@@ -227,6 +227,8 @@ public AbstractIndex getDataBlock() {
 
   private ReusableDataBuffer[] measureResusableDataBuffer;
 
+  private boolean readOnlyDelta;
```

Comment:
please add comment to describe this variable

Reply:
ok